res/css/views/dialogs/_InviteDialog.scss

```diff
@@ -318,6 +346,74 @@ limitations under the License.
     padding: 0;
 }
 
+.mx_InviteDialog_dialPad .mx_InviteDialog_dialPadField {
+    border-top: 0;
+    border-left: 0;
+    border-right: 0;
+    border-radius: 0;
+    margin-top: 0;
+    border-color: #6F7882;
```

Comment:
We might want to put that one in a variable, and check whether it needs to be different for the light and dark theme